dandi-info/config.yaml.j2

```diff
@@ -13,6 +13,8 @@ proxy:
       service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "https"
       service.beta.kubernetes.io/aws-load-balancer-connection-idle-timeout: "3600"
 hub:
+  extraEnv:
+    JUPYTERHUB_CRYPT_KEY: {{ os.stdout }}
```

Comment:
i had to add this for me to be able to do helm upgrade.

Reply:
I think i fixed a bug just before 1.0.0-beta.1 and that this now wont be needed.

It is a secret to encrypt/decrypt needed when storing auth_state in the jh database that is automaticallt generared now.

Can you describe why this was needed, did you get an error saying so when doing helm upgrade before any changes had been made to k8s for example, and what version did you experience this is?

Im asking as its relevant to ensure 1.0.0-beta.1 works as it should, and this should not be needed.

Reply:
this was needed, because `helm upgrade --install` would restart the hub pod, which would enter into a `CrashLoopBackOff` with this error: 

```
[I 2021-06-05 02:43:15.835 JupyterHub app:2349] Running JupyterHub version 1.3.0
[I 2021-06-05 02:43:15.836 JupyterHub app:2379] Using Authenticator: builtins.GitHubEnvAuthenticator
[I 2021-06-05 02:43:15.836 JupyterHub app:2379] Using Spawner: kubespawner.spawner.KubeSpawner-0.15.0
[I 2021-06-05 02:43:15.836 JupyterHub app:2379] Using Proxy: jupyterhub.proxy.ConfigurableHTTPProxy-1.3.0
[I 2021-06-05 02:43:15.837 JupyterHub app:1420] Loading cookie_secret from /srv/jupyterhub/jupyterhub_cookie_secret
auth_state is enabled, but encryption is not available: Encryption keys must be specified in JUPYTERHUB_CRYPT_KEY env
```

i am using the version `0.11.1-n479.hecce1f6b` haven't yet upgraded to 1.0.0-beta yet. it's just a placeholder in this PR for now.

Reply:
Are you building your own image? 0.11.1-n479 should be represented by this hub image, but I see that you have an old version of jupyterhub.

https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/ecce1f/images/hub/requirements.txt

Reply:
I think you won't need this in 1.0.0-beta.1 anyhow, and if you still do, that would be a bug that I want to fix.

Reply:
i will be testing out a new image soon, i will remove those two lines to check. the hub is running 1.0.0-beta now.

Reply:
and it does look like that requirements file means the fetch command should be available. i'll check all of the pieces :)

Reply:
it works without that piece. but i had to specify the version in the helm upgrade command. otherwise it somehow defaulted to JH app version 1.3.0.

as part of 1.0 is the proxy `secretToken` also similarly not required?

Reply:
If you don't specify the `--version` when installing a helm chart from a repo, you will get the latest stable release, which would be 0.11.1 at the moment.

proxy.secretToken is no longer required no! same with hub.cookieSecret but upgrade once with it set and then remove it - because otherwise it will generate a new one and cause the old cookies to malfunction.

Reply:
thanks @consideRatio - will check the changelogs :) but all of this is now working and i even pulled in yuvi's desktop proxy.